### ISSUE TITLE
Fish dark meat consumption in recent hours

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11017]
**Preferred term label:**
Fish dark meat consumption in recent hours

**Synonyms**

**Definition (free text, please give PubMed ID)**
tuna, mackerel, bonito, skip-jack, amberjack, bluefish, mahi-mahi, sardine, yellowtail, and herring

See https://www.ncbi.nlm.nih.gov/books/NBK499871/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Scombroid poisoning (histamine toxicity)

**Your nano-attribution (ORCID)**
